snowflake/data_access_sync_to_target.go

```diff
@@ -24,7 +24,150 @@ import (
 	"github.com/raito-io/cli-plugin-snowflake/common"
 )
 
-func (s *AccessSyncer) generateUniqueExternalId(ap *importer.AccessProvider, prefix string) (string, error) {
+type AccessToTargetSyncer struct {
+	ctx                           context.Context
```

Comment:
I'm really not in favour of storing the context as a member instead of passing it as first argument. 
Is their a specific reason you do it like this?